src/shared/lib/redirectHelpers.ts

```diff
@@ -168,3 +169,11 @@ export function handleIndexDO() {
         );
     }
 }
+
+export function handleEncodedRedirect() {
+    const encodedURL = getBrowserWindow().routingStore.location.query[
+        EncodedURLParam
+    ];
+    const decodedURL = atob(encodedURL);
+    getBrowserWindow().location.href = decodedURL;
```

Comment:
using the routingStore updateROute would allow us to replace history entry that contained the encoded thing.  with your solution, user could back button to it and then get redirected again, when they really might want to return to wherever they clicked the original bitly url.